### PR TITLE
Fix/back mobile change

### PR DESCRIPTION
**Pull request by LaurentMag**
# Description
Adapt back mobile to match database changes
So remove unused code, and renaming for clarity
Remove all sql script interacting with the database at API start

# Linked Issues
#585 

# Changes
What does it change ? fix mobile API to work with new database
Is is a breaking change ?
Is is a new feature ?
Is is a patch, fix, hotfix ?

# Screenshots
N/A

# Tests
How has it been tested ?

- [x] Manually
- [ ] Automated tests
- [ ] QA
- [ ] Other

# Additional information
N/A

### CODE REVIEW

File: yaki_backend/src/features/team/team.service.ts [hunk 1]
```diff
@@ -8,12 +8,11 @@ export class TeamService {
...
   }
 
-  getTeamByCaptainId = async (captain_id: number): Promise<TeamDtoIn> => {
-    const team = await this.teamRepository.getTeamByCaptainId(captain_id);
-    const teamDto: TeamDtoIn = new TeamDtoIn(team.team_id, team.team_captain_id, team.team_name);
-
-    if (teamDto !== undefined) {
-      return teamDto;
+  // return a team list managed by a selected captain id
+  getTeamByCaptainId = async (captain_id: number): Promise<TeamDtoIn[]> => {
```
Comment (profotoce59) [inline yaki_backend/src/features/team/team.service.ts:12]: rename to getTeamsByCaptainId

File: yaki_backend/src/features/team/team.repository.ts [hunk 2]
```diff
@@ -2,8 +2,6 @@ import {Client, QueryResult} from "pg";
 import {TeamDtoIn} from "./team.dtoIn";
 
 export class TeamRepository {
-
-
   getTeamByCaptainId = async (captainId: number) => {
```
Comment (profotoce59) [inline yaki_backend/src/features/team/team.repository.ts:5]: rename to getTeamsByCaptainId

File: yaki_backend/src/features/team/team.service.ts (hunk 1 above)
Comment (LaurentMag) [reply yaki_backend/src/features/team/team.service.ts:12]: renamed

File: yaki_backend/src/features/team/team.repository.ts (hunk 2 above)
Comment (LaurentMag) [reply yaki_backend/src/features/team/team.repository.ts:5]: renamed

**Pull request closed without merging**